Nuro/Utilities/Extensions/Date+Ext.swift

```diff
@@ -25,8 +25,9 @@ extension Date {
         let hour = calendar.component(.hour, from: date)
         let minute = calendar.component(.minute, from: date)
         let second = calendar.component(.second, from: date)
+        let nanoSecond = calendar.component(.nanosecond, from: date)
 
-        let timestamp = "Nuro_\(year)\(month)\(day)_\(hour)\(minute)\(second)"
+        let timestamp = "Nuro_\(year)\(month)\(day)_\(hour)\(minute)\(second)\(nanoSecond)"
```

Comment:
Nice extra step. Nano second is okay, but for this case, using second is actually enough. Great job!